File: Quacking/solve/java/QuackingSolver.java
import java.io.FileReader;
import java.io.FileWriter;

import java.util.List;
import java.util.ArrayList;

import org.topcoder.TopcoderReader;
import org.topcoder.TopcoderWriter;
import org.topcoder.TypeRef;

public class QuackingSolver {
    public static void main(String[] args) {
    try {
        TopcoderReader reader = new TopcoderReader(new FileReader(args[0]));
        String s = (String) reader.next(String.class);
        reader.close();

        Quacking solver = new Quacking();
        TopcoderWriter writer = new TopcoderWriter(new FileWriter(args[1]));
        writer.write(solver.quack(s));
        writer.close();
    } catch (Exception err) {
        err.printStackTrace(System.err);
    }
    }
}

File: Quacking/solve/cpp/QuackingSolver.cpp
#include <fstream>
#include <topcoder>
#include "Quacking.cpp"
namespace tc = TopCoder;

int main(int argc, char const *argv[]) {
  try {
    std::ifstream ifs(argv[1]);
    string s; tc::read(ifs, s);
    ifs.close();

    std::ofstream ofs(argv[2]);
    Quacking solver;
    tc::show(ofs, solver.quack(s));
    ofs.close();
  } catch (std::exception &e) {
    std::cerr << e.what() << std::endl;
  }
  return 0;
}
